lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -375,7 +377,7 @@ def _on_refresh(self, event):
                         dashboards=self._dashboards,
                         metrics_scrape_jobs=self._scrape_jobs,
                         log_slots=self._log_slots,
-                        subordinate=self._charm.meta.subordinate,
+                        subordinate=None,
```

Comment:
if we make subordinate a defaulted field, we can remove it from here AND detect if someone is attempting to use it